### ISSUE TITLE
Map folios csv import/export not functional

### ISSUE THREAD

**Issue by dchiller** [DDMAL/cantus#661]
- The map folios csv backup button exports a csv that has duplicated lines in it. Notice below how image uri's are repeated: 
<img width="293" alt="image" src="https://user-images.githubusercontent.com/11023634/208702283-61521caa-f064-41d2-b48e-6bdad47875a9.png">

- Dragging and dropping a csv backup does not seem to instigate any response on the map folios page.

**Comment by dchiller** [comment]
Testing this on staging with Release 3.2-0.11.3 and this seems still not to work when uploading a backup from production. Reopening...

**Comment by dchiller** [comment]
Closing again...

This actually is functional, but in #691 we adjusted both the export and import process for folio mapping backups. What doesn't work are the exported csvs from the old process **with** the new import process, because the old import process creates two lines per IIIF canvas even for single-folio images. To fix, I just remove those duplicated lines in the csv file manually and use the upload feature. Because in the future the backup csv's will be generated by the new process, this will never again be an issue (unless the backup/generation process changes again).
